### PR TITLE
Feature/aa 1245 add region markups endpoints

### PR DESCRIPTION
**Pull request by DaniyarAskarov**
Endpoints for region markups' CRUD methods

### CODE REVIEW

**Comment by DaniyarAskarov** [comment]
Issue: https://github.com/happy-travel/agent-app-project/issues/1245

**Comment by DaniyarAskarov** [comment]
Related PR's : https://github.com/happy-travel/edo/pull/1559

File: Api/Services/Markups/AdminMarkupPolicyManager.cs [hunk 1]
```diff
@@ -34,13 +34,13 @@ public class AdminMarkupPolicyManager : IAdminMarkupPolicyManager
         {
             _context = context;
             _templateService = templateService;
-            _dateTimeProvider = dateTimeProvider;;
+            _dateTimeProvider = dateTimeProvider; ;
```
Comment (AlexeyShok) [inline Api/Services/Markups/AdminMarkupPolicyManager.cs:37]: Extra semicolon

File: Api/Services/Markups/AdminMarkupPolicyManager.cs [hunk 2]
```diff
@@ -204,29 +206,32 @@ private static Result<MarkupPolicyData> GetPolicyData(MarkupPolicy policy)
...
         }
-        
-        
-        private async Task<Result<SubjectMarkupScopeTypes>> GetAgentMarkupScopeType(string locationId)
+
+
+        private async Task<Result<SubjectMarkupScopeTypes>> GetAgentMarkupScopeType(MarkupPolicySettings settings)
         {
-            var (_, isFailure, value, error) = await _mapperClient.GetSlimLocationDescription(locationId);
+            if (settings.LocationScopeType != SubjectMarkupScopeTypes.NotSpecified)
```
Comment (AlexeyShok) [inline Api/Services/Markups/AdminMarkupPolicyManager.cs:219]: if (settings.LocationScopeType == SubjectMarkupScopeTypes.Region)

**Review by AlexeyShok — APPROVED** [review]

**Review by kirillta — APPROVED** [review]

**Pull request merged**